dicom_numpy/combine_slices.py

```diff
@@ -233,7 +234,13 @@ def _slice_spacing(slice_datasets):
         slice_positions_diffs = np.diff(sorted(slice_positions))
         return np.mean(slice_positions_diffs)
     else:
-        return 0.0
+        dataset = slice_datasets[0]
+        if hasattr(dataset, 'SliceThickness'):
+            return dataset.SliceThickness
```

Comment:
I think the `SliceThickness` means something different than the `SpacingBetweenSlices`. E.g., for an MR slice, you could sample 1mm worth of tissue yet have the slices be 3mm apart. Thus, I think we should check for the `SpacingBetweenSlices` if present, and if not, return 0.0.

Reply:
You are probably right, I wasn't fully aware of this difference. I will check some data and definitions and fix this accordingly

Reply:
For future reference, the `SpacingBetweenSlices` is indeed the correct spacing and not `SliceThickness`, as can be checked by the series from here:
https://www.dicomlibrary.com/meddream/?study=1.2.826.0.1.3680043.8.1055.1.20111103111148288.98361414.79379639